### PR TITLE
Generation from interface aliases

### PR DESCRIPTION
**Pull request by zcolleen**
As described in https://github.com/gojuno/minimock/issues/106 . Should be updated and merged after https://github.com/hexdigest/gowrap/pull/89

### CODE REVIEW

File: cmd/minimock/minimock.go [hunk 1]
```diff
@@ -383,6 +384,7 @@ func processArgs(args []string, stdout, stderr io.Writer) (*options, error) {
 
 	fs.BoolVar(&opts.noGenerate, "g", false, "don't put go:generate instruction into the generated code")
 	fs.StringVar(&opts.suffix, "s", "_mock_test.go", "mock file suffix")
+	fs.BoolVar(&opts.generateFromAliases, "from-aliases", false, "generate mocks from interface aliases as well")
```
Comment (zcolleen) [inline cmd/minimock/minimock.go:387]: discussable if we should ad this feature as an option or make it as default behaviour

Comment (hexdigest) [reply cmd/minimock/minimock.go:387]: Hey, Simon

I think this should be default behavior.

File: internal/types/interface.go [hunk 2]
```diff
@@ -57,8 +56,128 @@ func FindAllInterfaces(p *ast.Package, pattern string) []InterfaceSpecification
...
+	fs := token.NewFileSet()
+	srcAst, err := pkg.AST(fs, srcPkg)
+	if err != nil {
+		return nil, err
+	}
+
+	return srcAst, nil
+}
+
+func findTypeSoecInPackage(p *ast.Package, name string) (typeSpec *ast.TypeSpec, imports []*ast.ImportSpec) {
```
Comment (hexdigest) [inline internal/types/interface.go:135]: Soec typo

File: cmd/minimock/minimock.go (hunk 1 above)
Comment (zcolleen) [reply cmd/minimock/minimock.go]: fixed

File: internal/types/interface.go (hunk 2 above)
Comment (zcolleen) [reply internal/types/interface.go]: fixed

**Review by hexdigest — APPROVED** [review]

**Pull request merged**